test/e2e/commands.js

```diff
@@ -114,7 +114,20 @@ function testContainsOrNotExists(target, value) {
   }
 }
 
+function testMultipleForContainsOrNotExists(testObjects = []) {
+  testObjects.forEach((testObject) => {
+    const {target = '', value = '', stripHtml = false} = testObject;
+
+    if (stripHtml) {
+      cy.testContainsOrNotExists(target, cy.helpers.stripHtml(value));
+    } else {
+      cy.testContainsOrNotExists(target, value);
+    }
+  });
```

Comment:
```suggestion
  testObjects.forEach(({ target = '', value = '', stripHtml = false }) => {
    if (stripHtml) value = cy.helpers.stripHtml(value)
    cy.testContainsOrNotExists(target, value);
  });
```

You can directly destructure function parameters in situ.

Otherwise the parameter brackets here are not needed
```js
testObjects.forEach((testObject) => {
``` 
```js
testObjects.forEach(testObject => {
``` 
Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#syntax


Perfer return early over an `else` clause:
```js
    if (stripHtml) {
      cy.testContainsOrNotExists(target, cy.helpers.stripHtml(value));
    } else {
      cy.testContainsOrNotExists(target, value); // this is the main use-case in the last else
    }
```
```js
    if (stripHtml) {
      cy.testContainsOrNotExists(target, cy.helpers.stripHtml(value));  // alternate use-case
      return;
    }
    cy.testContainsOrNotExists(target, value);  // main use-case
```